wordpress-mobile/WordPress-Android
# fix wordpress-mobile/WordPress-Editor-Android#7: editor onboarding

Fixes https://github.com/wordpress-mobile/WordPress-Editor-Android/issues/7

Message will appear on the Post List if the app is not a fresh install and if the user have visual editor enabled.

First idea was to show it on launch but I don't like how it can interfere if you open the app for specific purpose (read your notifications for instance).

I reused the Stats promo dialog style for consistency, and the same message shown in the iOS app.

To test:

* Do you get the dialog on app upgrade iff visual editor is enabled? (YES!)
* Do you get the dialog on a fresh install? (NO!)
* Make sure Stats dialog is correctly displayed.

Needs review: @aforcier 

![screenshot-2016-03-04_09 30 37 522](https://cloud.githubusercontent.com/assets/40213/13521818/c89c1eea-e1eb-11e5-87c6-69a394d2bf38.png)
